contributors/devel/sig-node/cherry-picks.md

```diff
@@ -0,0 +1,115 @@
+# SIG Node Cherry-Pick Management Process
+
+Kubernetes supports multiple release versions simultaneously. How do new fixes
+make it into earlier versions, when we only develop one version at a time?
+
+We [cherry-pick] them!
+
+[cherry-pick]: https://git-scm.com/docs/git-cherry-pick
+
+## Release timeline
+
+SIG Release regularly (on a ~monthly cadence) ships [patch releases]. These are
+intended for critical bug fixes to the latest minor version, such as addressing
+security vulnerabilities, fixes to problems affecting a large number of users,
+severe problems with no workaround, and blockers for products based on
+Kubernetes.
+
+Every release, the Release Team publishes a list of [target dates] with
+**cherry-pick deadlines**. SIG Node's responsibility is to ensure that all
+patches that we want cherry-picked to previous releases must be [triaged],
+prioritized, LGTM'd, and approved by the deadline.
+
+[patch releases]: https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md#patch-releases
+[target dates]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#upcoming-monthly-releases
+[triaged]: triage.md
+
+## Monthly cherry-pick process
+
+Throughout the release cycle, merged bugfixes should be evaluated for inclusion
+as cherry-picks. Any bugs marked **critical-urgent** should be considered. Some
+**important-soon** bugs may also be considered if they are sufficiently
+contained and meet the criteria above. **Only** bug fixes should be considered for backports.
+
+To create a cherry-pick PR, you can use the `./hack/cherry_pick_pull.sh`
+script. Read through the [cherry pick documentation] for details on that
+process.
+
+The week of the cherry-pick deadline, create a Slack thread ([example
```

Comment:
I'd say we need to have a process for before the last week. Let's mention that for questionable cherry-picks, those can be discussed in Slack or in SIG Node meeting

Reply:
The target audience of this doc is the Node person stewarding the cherry-picks, not authors proposing cherry-picks. I agree that people should raise controversial cherry-picks early, but it's not the responsibility of the cherry-pick steward.

Reply:
Added a paragraph making this audience/purpose more clear.